src/app/components/shared/modal-post-reply/modal-post-reply.component.html

```diff
@@ -10,6 +10,9 @@
       [allowExpansion]="false"
     ></ksi-discussion-thread-posts>
     <quill-editor class="reply-editor" formControlName="content"></quill-editor>
+    <div class="post-size d-flex justify-content-end"
+      [class]="{'text-danger': form.controls.content.value?.length > 8000}"
```

Comment:
Mohl bys z toho prosím udělat konstantu? Podobně jako je THREAD_NAME_MAX_LENGTH, jenom asi ne statickou (jinak to nebude fungovat v HTML template, pokud se nepletu)